### ISSUE TITLE
tJasperServerFileList resource type parameter

### ISSUE THREAD

**Issue by liviutudorMTD** [jlolling/talendcomp_tJasperServerFile#6]
I noticed that there are some (older) open threads on talend or jasper support forums of people that are wondering why tJasperServerFileList does not return anything. I think that this is because the some are interested in other resource types than the default hardcoded value of the query string parameter that is "file": type=file. I think would be nice to have a parameter exposed for this query string parameter in the component just for example as it is recursive and filter. This would allow to use tJasperServerFileList component to fetch for example: "adhocDataView" resource type.

Great work BTW.

**Comment by jlolling** [comment]
Hi, I do not have understood what you mean. Could you please elaborate your thoughts a bit more in deep?

**Comment by jlolling** [comment]
I agree with you. In the former release of the JasperServer it was not possible to download anything else than contentResource typed files. I will check that and allow also other types.
